docs/pages/accounts/sso.mdx

```diff
@@ -21,6 +26,34 @@ Expo SSO supports the following identity providers:
 
 We implement the [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html) specification and are working to verify additional compatible identity providers. If you use another identity provider and are interested in SSO, [let us know](https://expo.dev/contact).
 
+## Enable SSO
+SSO is enabled by creating SSO configuration for the Organization.
+
+<Step label="1">
+	Log in as the Organization account owner, select your Organization Account, then go to Organization Settings > Overview.
+</Step>
+
+<Step label="2">
+	Click the ‘Start’ button next to the ‘Create SSO configuration for account’ option.
+</Step>
+
+<Step label="3">
+	Enter the configuration details for your IdP in the form, with the information from setting up your IdP:
+  - your client ID
+  - your client secret
+  - your IdP subdomain/tenant ID, if needed. Check the Issuer field help for what to enter.
```

Comment:
```suggestion
  - Client ID
  - Client secret
  - IdP subdomain/tenant ID, if needed. Check the Issuer field for help with what to enter.
```

Let's omit the word "your" here since it's obvious to the user that they have to enter their credentials.

Reply:
Deleted your

Changed 
  Check the Issuer field for help with what to enter
to:
  Click the **?** icon above the Issuer field for help with what to enter.

As I'm trying to express click the help icon for help, without using the word help twice

Reply:
Thank you for making it more clear by updating it 🙏